### ISSUE TITLE
APK request

### ISSUE THREAD

**Issue by kaveenkumar** [farmaker47/Yamnet_classification_project#3]
Hi, thanks for the project.

Is there a compiled Android package file (.apk), readily available to install and test on a mobile phone?

Thanks!

**Comment by farmaker47** [comment]
Hi,
No available .apk
Can you build the project with Android studio?

**Comment by kaveenkumar** [comment]
Thanks for the reply.
I am trying to build the project on Android studio, however, I am facing few issues with JDK version and Gradle version compatibility. Do you have like a strict version requirement of this somewhere?

**Comment by kaveenkumar** [comment]
Building the project out-of-the-box from this GitHub repo throws an error, which was fortunately solved by [stackoverflow](https://stackoverflow.com/a/69179997/12584995).. however I have a different error now and assuming its because of JDK version on my machine

**Comment by farmaker47** [comment]
Yes, this is a 4 years old project! Gradle and AS version may not be compatible.
The only thing you can do is start a new project and copy paste there all the java/kotlin files and assets from here
